test/fix-15509.html

```diff
@@ -0,0 +1,58 @@
+<html>

```

Comment:
Please use a more specific name for the test case. You create a single test file for this algorithm. Run
```ts
npm run mktest testName
```